.github/ISSUE_TEMPLATE/if-plugin-registry-form-template.md

```diff
@@ -0,0 +1,32 @@
+---
+name: IF Plugin Registry Form template
+about: Use this template to submit an IF plugin for registry
+title: "[PLUGIN REGISTRY]"
+labels: ''
+assignees: ''
+
+---
+
+## Plugin Name
+<!--- How do you call your plugin? -->
+
+## Plugin GitHub URL
+<!--- URL -->
+
+## Plugin Registry Basic Requirements
+- [ ] My plugin executes a demo manifest yml with no errors or crashes.
+- [ ] My plugin has README file.
+- [ ] My plugin code includes a sample manifest .yaml file.
+- [ ] My plugin's code contain unit tests that cover >=80% of the plugins code.
+- [ ] All unit tests that test my plugin are passing.
+- [ ] All other unit tests in the if-models repo are passing after my code is integrated into it.
+- [ ] My plugin's code have no security vulnerabilities. 
+
+## Define, in your own words, what is(are) the Requirement(s) of your plugin
+<!--- I.E what is the basic functionality it must demonstrate. -->
+
+## Please provide citations, links and references that support the validity of the implementation of your plugin
+<!--- I.E provide reference to sources explaining how this implementation meets the requirement described above. -->
+
+## Please describe the test cases covered by your plugin's dedicated unit tests
```

Comment:
```suggestion
```

Again, let's handle this elsewhere

Reply:
again, I don't mind. just think we need 1 place where all info (that is - the answers to the questions in this form) for all registered plugins reside in.